libraries/botbuilder-dialogs/tests/skillDialog.test.js

```diff
@@ -194,202 +212,197 @@ describe('SkillDialog', function() {
     });
 
-    it('resumeDialog() should call repromptDialog()', async () => {
+    it('resumeDialog() should call repromptDialog()', async function () {
         const adapter = new TestAdapter(/* logic param not required */);
         const context = new TurnContext(adapter, { type: ActivityTypes.Message, id: 'activity-id' });
         context.turnState.set(adapter.OAuthScopeKey, DEFAULT_OAUTHSCOPE);
-        const dialog = new SkillDialog({} , 'SkillDialog');
-        
+        const dialog = new SkillDialog({}, 'SkillDialog');
+
         let repromptDialogCalled = false;
         dialog.repromptDialog = () => {
             repromptDialogCalled = true;
         };
-        
+
         const result = await dialog.resumeDialog(context, {});
-        ok(repromptDialogCalled, 'sendToSkill not called');
-        strictEqual(result, Dialog.EndOfTurn);
+        assert.ok(repromptDialogCalled, 'sendToSkill not called');
+        assert.strictEqual(result, Dialog.EndOfTurn);
     });
 
-    describe('(private) validateBeginDialogArgs()', () => {
-        it('should fail if options is falsy', () => {
+    describe('(private) validateBeginDialogArgs()', function () {
+        it('should fail if options is falsy', function () {
             const activity = {
                 type: ActivityTypes.Message,
-                text: 'Hello SkillDialog!'
+                text: 'Hello SkillDialog!',
             };
             const dialog = new SkillDialog({}, 'SkillDialog');
             const validatedArgs = dialog.validateBeginDialogArgs({ activity });
             const validatedActivity = validatedArgs.activity;
-            strictEqual(validatedActivity.type, ActivityTypes.Message);
-            strictEqual(validatedActivity.text, 'Hello SkillDialog!');
-        });
-
-        it('should fail if options is falsy', () => {
-            try {
-                const dialog = new SkillDialog({}, 'SkillDialog');
-                dialog.validateBeginDialogArgs();
-            } catch (e) {
-                typeErrorValidator(e, 'Missing options parameter');
-            }
+            assert.strictEqual(validatedActivity.type, ActivityTypes.Message);
+            assert.strictEqual(validatedActivity.text, 'Hello SkillDialog!');
         });
 
-        it('should fail if dialogArgs.activity is falsy', () => {
-            try {
-                const dialog = new SkillDialog({}, 'SkillDialog');
-                dialog.validateBeginDialogArgs({});
-            } catch (e) {
-                typeErrorValidator(e, '"activity" is undefined or null in options.');
-            }
+        it('should fail if options is falsy', function () {
+            const dialog = new SkillDialog({}, 'SkillDialog');
+            assert.throws(() => dialog.validateBeginDialogArgs(), new TypeError('Missing options parameter'));
         });
 
-        it('should fail if dialogArgs.activity.type is not "message" or "event"', () => {
```

Comment:
Note: This method was removed. When "fixed" so that it throws when expected, it doesn't actually throw. This is because [it doesn't validate within the activity](https://github.com/microsoft/botbuilder-js/blob/77d8fb7893881b3072b3c148cb256409955e8b96/libraries/botbuilder-dialogs/src/skillDialog.ts#L231) and [neither does .NET](https://github.com/microsoft/botbuilder-dotnet/blob/395c8bd1365dc042600ac8fe8546afc6f6eafaf4/libraries/Microsoft.Bot.Builder.Dialogs/SkillDialog.cs#L218)